com.unity.shadergraph/Editor/Data/Nodes/Mesh Deformation/ComputeDeformNode.cs

```diff
@@ -0,0 +1,148 @@
+using UnityEngine;
+using UnityEditor.Graphing;
+using UnityEditor.ShaderGraph.Internal;
+
+namespace UnityEditor.ShaderGraph
+{
+    [Title("Mesh Deformation", "Compute Deformation")]
+    class ComputeDeformNode : AbstractMaterialNode, IGeneratesBodyCode, IGeneratesFunction, IMayRequirePosition, IMayRequireNormal, IMayRequireTangent, IMayRequireVertexID
+    {
+        public const int kPositionOutputSlotId = 0;
+        public const int kNormalOutputSlotId = 1;
+        public const int kTangentOutputSlotId = 2;
+
+        public const string kOutputSlotPositionName = "Position";
+        public const string kOutputSlotNormalName = "Normal";
+        public const string kOutputSlotTangentName = "Tangent";
+
+        public ComputeDeformNode()
+        {
+            name = "Compute Deformation";
+            UpdateNodeAfterDeserialization();
+        }
+
+        public sealed override void UpdateNodeAfterDeserialization()
+        {
+            AddSlot(new Vector3MaterialSlot(kPositionOutputSlotId, kOutputSlotPositionName, kOutputSlotPositionName, SlotType.Output, Vector3.zero, ShaderStageCapability.Vertex));
+            AddSlot(new Vector3MaterialSlot(kNormalOutputSlotId, kOutputSlotNormalName, kOutputSlotNormalName, SlotType.Output, Vector3.zero, ShaderStageCapability.Vertex));
+            AddSlot(new Vector3MaterialSlot(kTangentOutputSlotId, kOutputSlotTangentName, kOutputSlotTangentName, SlotType.Output, Vector3.zero, ShaderStageCapability.Vertex));
+            RemoveSlotsNameNotMatching(new[] { kPositionOutputSlotId, kNormalOutputSlotId, kTangentOutputSlotId });
+        }
+
+        protected override void CalculateNodeHasError()
+        {
+#if !ENABLE_COMPUTE_DEFORMATIONS
+            owner.AddSetupError(guid, "ENABLE_COMPUTE_DEFORMATIONS define needs to be enabled in Player Settings for Compute Deformation node to work");
+            hasError = true;
+#endif
+#if !HYBRID_RENDERER_0_4_0_OR_NEWER
+            owner.AddSetupError(guid, "Could not find 0.4.0 version or newer of Hybrid Renderer installed");
```

Comment:
What about:
This node requires the Hybrid Renderer package to function properly.